extensions-core/multi-stage-query/src/main/java/org/apache/druid/msq/indexing/DurableStorageCleanerConfig.java

```diff
@@ -19,59 +19,31 @@
 
 package org.apache.druid.msq.indexing;
 
-import com.fasterxml.jackson.annotation.JsonCreator;
 import com.fasterxml.jackson.annotation.JsonProperty;
-import com.google.common.base.Preconditions;
+
+import javax.validation.constraints.Min;
 
 public class DurableStorageCleanerConfig
 {
 
-  private static final long DEFAULT_INITIAL_DELAY_SECONDS = 86400L;
-  private static final long DEFAULT_DELAY_SECONDS = 86400L;
-
   /**
    * Whether the {@link DurableStorageCleaner} helper should be enabled or not
    */
   @JsonProperty
-  private final boolean enabled;
-
-  /**
-   * Initial delay in seconds post which the durable storage cleaner should run
-   */
-  @JsonProperty
-  private final long initialDelaySeconds;
+  public boolean enabled = false;
```

Comment:
The config properties could be private

Reply:
Yeah, it should be private but I think that can be done as part of a follow-up PR.